api-gateway/default.conf

```
@@ -165,6 +170,26 @@ server {
         add_header Content-Type application/json;
     }
 
+    # Question Service API
+    location /api/question/ {
```

```suggestion
    location /api/question {
```
otherwise nginx will redirect POST /api/question to GET /api/question if there is a / behind (weird but it is what it is)